### DISCUSSION TITLE
Preparing the v7.2.2 release

### DISCUSSION THREAD

**Discussion by vmiklos** [vmiklos/plees-tracker#194] [General]
Hi,

I plan to release v7.2.2 towards the end of the next week. It currently contains a new English-only string.

@sguinetti @fabianski7 may I ask you to update the translations you already created in the past?

Test APK here: https://share.vmiklos.hu/plees-tracker/app-debug-v7.2.2-rc1.apk

Release notes: https://github.com/vmiklos/plees-tracker/blob/master/NEWS.adoc

Thanks,

Miklos

**Comment by fabianski7** [comment]
Thanks for the notification. please check if https://github.com/vmiklos/plees-tracker/pull/195 is correct

**Comment by sguinetti** [comment]
Hello. I update new strings in PR #203.
